timemachine/ff/handlers/bonded.py

```diff
@@ -76,6 +76,14 @@ def static_parameterize(params, smirks, mol):
         if len(mol_params) == 0:
             mol_params = params[:0]  # empty slice with same dtype, other dimensions
             bond_idxs = np.zeros((0, 2), dtype=np.int32)
+
+            expected_num_bonds = mol.GetNumBonds()
+            assigned_num_bonds = len(bond_idxs)
```

Comment:
Nit: its a bit confusing that this is dynamic but is always zero per line 78

Reply:
Is this suppose to be unindented??

Reply:
Oof, good catch -- this shouldn't have been inside the `if len(mol_params) == 0` branch! Reordered and fixed in https://github.com/proteneer/timemachine/pull/742/commits/727449df41ff10eac22849c29b9ca3ffba548b80